deerlab/dipolarmodel.py

```diff
@@ -69,10 +69,10 @@ def dipolarmodel(t, r, Pmodel=None, Bmodel=bg_hom3d, npathways=1, harmonics=None
     if not isinstance(harmonics,list) and harmonics is not None:
         raise TypeError('The harmonics must be specified as a list of integer values.')
     if experiment is not None:
-        npathways = experiment.npathways
         if not isinstance(experiment,ExperimentInfo): 
-            raise TypeError('The experiment must be a valid deerlab.ExperimentInfo object.')
+            raise TypeError('The experiment must be a valid deerlab. object.')
```

Comment:
Why removing the `ExperimenInfo` from `deerlab.ExperimenInfo`? Now the exception string makes no sense.

Reply:
Fixed